config/apache2-ironic-api.conf.j2

```diff
@@ -0,0 +1,47 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+Listen 6385
+
+<VirtualHost *:6385>
```

Comment:
There's an environment variable for LISTEN_ALL_INTERFACES or not, shouldn't we change this VirtualHost to use `{{ env.IRONIC_IP }}:6385` if that's false?

Reply:
Fixed!. Thanks